webapps-schema/src/main/resources/schema/elasticsearch/create/index/identity-tenants.json

```diff
@@ -0,0 +1,25 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "tenantKey": {
+        "type": "long"
+      },
+      "tenantId": {
+        "type": "keyword"
+      },
+      "name": {
+        "type": "keyword"
+      },
+      "entityKey": {
+        "type": "long"
+      },
+      "entityType": {
+        "type": "keyword"
+      }
```

Comment:
What does the entityKey+entityType reference point to? I'm a bit confused because I would either expect these documents to describe a named tenant entity or a "tenant annotation" on other entities, which I wouldn't expect to contain a denormalized tenant name.

Reply:
In this index definition these fields aren't really relevant, in the Zeebe engine there is a TenantRecord, the fields exist there as a way for the broker to send commands to add (or remove) an entity (member) from a tenant. The field stored in ES/OS would actually be a collection of all the entities (members) assigned to this specific tenant.